tests/virtualization/xen/hotplugging.pm

```diff
@@ -40,9 +40,10 @@ sub run_test {
             if (get_var('VIRT_AUTOTEST') && (check_var('SYSTEM_ROLE', 'kvm') || check_var('HOST_HYPERVISOR', 'kvm'))) {
                 $interface_model_option = '--model virtio';
                 script_run "brctl addbr br0; ip link set dev br0 up", 60 if ($sles_running_version eq '11' && $sles_running_sp eq '4');
-                script_run "ssh root\@$guest modprobe acpiphp", 60 if ($guest =~ /^sles-11-sp4.*$/img);
-                record_soft_failure 'bsc#1167828 Bridge network interface br0 hotplugging does not work with this ' . $guest if ($guest =~ /^sles-11-sp4.*$/img);
-                $self->{test_results}->{$guest}->{"bsc#1167828 Bridge network interface br0 hotplugging does not work with this $guest"}->{status} = 'SOFTFAILED' if ($guest =~ /^sles-11-sp4.*$/img);
+                if ($guest =~ /^sles-11-sp4.*$/img) {
+                    script_run "ssh root\@$guest modprobe acpiphp", 60;
+                    record_info('Workaround bsc#1167828 Bridge network interface br0 hotplugging does not work with SLE 11-SP4 guest');
```

Comment:
I just checked the bug, it is won't fix status now. So yes, we should remove the softfail. But the new record info in L45 should also be re-phased/dropped because the bug is closed now, it is not workaround any more, but that it is necessary setup now.

Reply:
Yes. It is closed. But the test needs workaround to function otherwise it will fail.

Reply:
Yes and no. 
IMHO, the bug is closed because our product does not want to support it that way. If using this info "workaround bsc#xx", it will lead people to "the bug is still open and valid, just that it is not fixed yet so we workaround it". You can think about why in L88 you used "record_info skip" rather than "record_info 'error, it is not supported for 11sp4'". Quite similar.

I explained so much because I want to make one point clear that only when bug is valid(open->confirmed->fixed), we can call the necessary setup for automation as workaround. Or it is just necessary setup or test step, just as other regular  steps. This rule will always work for future record_info or hints or final test status that we show to reviewers like qa, release mgrs, team lead or anyone else.

Reply:
I remove the "workaround" word, and explain a bit more about the situation. Done.